README.md

```diff
@@ -49,69 +49,8 @@ terraform init -upgrade
 to upgrade to the latest stable version of the vra7 provider. See the [Terraform website](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
 for more information on provider upgrades, and how to set version constraints on your provider.
 
-Migrating from previous versions to version 1.0.0, issues fixed and enhancements
```

Comment:
Good to see this 1.0.0 section removed. Should there be a comment added somewhere in the README about the vra7.5/3.0 vs. vra7.4/2.0 versioning?

Reply:
Done!